js/components/menuList/index.js

```diff
@@ -9,20 +8,38 @@ import ImageCollections from 'material-ui/svg-icons/image/collections';
 import NavigationMenu from 'material-ui/svg-icons/navigation/menu';
 import styles from './menuList.scss';
 
-const MenuList = () => {
-  return (
-    <div>
-      <div className={styles['sandwich-button']}>
-        <NavigationMenu color='#000'/>
+class MenuList extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isListOpened: true
+    };
+    this.handleSandwichClick = this.handleSandwichClick.bind(this);
+  }
+
+  handleSandwichClick() {
+    this.setState({isListOpened: !this.state.isListOpened});
```

Comment:
it this case will be better use function to change state, because you set opposite value to field ```isListOpened: !this.state.isListOpened```